TPP.Persistence.MongoDB/Repos/UserRepo.cs

```diff
@@ -123,6 +135,8 @@ public async Task<User> RecordUser(UserInfo userInfo)
             try
             {
                 await Collection.InsertOneAsync(document: user);
+                if (_defaultOperators.Contains(user.SimpleName))
+                    await SetRoles(user, new HashSet<Role> { Role.Operator });
```

Comment:
you can already specify the roles in the User object's constructor instead of adding them here retroactively